Modules/Synchronization/src/Synchronization.Application/SyncRuns/Commands/FinalizeSyncRun/FinalizeExternalEventSyncSyncRunCommandValidator.cs

```diff
@@ -20,7 +20,7 @@ public class EventResultValidator : AbstractValidator<FinalizeExternalEventSyncS
         public EventResultValidator()
         {
             RuleFor(i => i.ExternalEventId).ValidId<FinalizeExternalEventSyncSyncRunCommand.ExternalEventResult, ExternalEventId>();
-            RuleFor(i => i.ErrorCode).MaximumLength(50);
+            RuleFor(i => i.ErrorCode).MaximumLength(100);
```

Comment:
is this enough? Don't you have to add a DB migration?

Reply:
Good catch! Fixed

Reply:
I just tested it manually, and a 100 char error code works.